test/clj/cider/nrepl/middleware/info_test.clj

```diff
@@ -424,6 +424,17 @@
       (let [response (session/message {:op "eldoc" :symbol "." :ns "user"})]
         (is (= (:status response) #{"done"}))))
 
+    (testing "java interop method with multiple classes"
+      (let [response (session/message {:op "eldoc" :symbol ".length" :ns "cider.nrepl.middleware.info-test"})]
+        (is (= (:class response)
+               "java.lang.String, java.lang.StringBuffer, java.lang.CharSequence, java.lang.StringBuilder"))
```

Comment:
Done. Some unrelated tests failed, `cider.nrepl.print-method-test/print-idrefs` and `cider.nrepl.middleware.debug-integration-test`